build.sh

```diff
@@ -3,12 +3,14 @@
 # this script is use to build basic_bot itself.  Not included in
 # the files that `bb_create` creates.
 
-rm -Rf src/basic_bot/created_files/webapp/node_modules
-rm -Rf src/basic_bot/created_files/webapp/package-lock.json
 
 # fail on any error
 set -e
 
+echo "Installing npm dependencies locally..."
+cd src/basic_bot/created_files/webapp/ && npm install
+cd ../../../..
```

Comment:
For building, we really need to know if the webapp can build.  Should be safe to assume that you have npm on your development machine and that ./build.sh is only run from a development env